routes.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type route struct {
+	Name        string
+	Method      string
+	Pattern     string
+	HandlerFunc http.HandlerFunc
+}
+
+var routes = []route{
+	route{
+		"hello",
+		"GET",
+		"/hello/{name}",
+		helloHandler,
+	},
+	route{
+		"initiateTranscriptionJob",
+		"DELETE",
+		"/add_job",
+		initiateTranscriptionJobHandler,
+	},
```

Comment:
meant that health is merged into master. as long as the method gets included, lgtm